packages/esm-system-admin-app/src/change-password-modal/change-password-modal.scss

```diff
@@ -0,0 +1,9 @@
+.input-group {
+  :global(.cds--label) {
+    margin-top: 1rem;
+  }
+
+  :global(.cds--text-input) {
+    background-color: white;
+  }
```

Comment:
Use [Layer](https://react.carbondesignsystem.com/?path=/docs/components-layer--overview) instead of this override.